calculator.js

```diff
@@ -0,0 +1,83 @@
+//array of numbers for our buttons
+var array = [1, 2, 3, 4, 5, 6, 7, 8, 9, "+", "-", "*"]
+
+//makes the Input
+
+function makeInput() {
+    var mkInput = document.createElement("input");
+    mkInput.className = "cal-input";
+    mkInput.id = "DtaInput";
+    document.body.appendChild(mkInput);
+}
+
+function br() {
+    document.createElement('br');
+    return br
+}
+//this is not used
+//var num = event.target.textContent;
+
+
+//this puts text inside of our input when we do our click event.
+function handleButton(button) {
+    debugger;
```

Comment:
Please either comment out or remove debugger statements when you submit the assignment.